### PR TITLE
refactor: extract getAppInfo that can be extend

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

baseLoader

### CODE REVIEW

File: lib/loader/egg_loader.js [hunk 1]
```diff
@@ -120,6 +119,26 @@ class EggLoader {
   }
 
   /**
+   * Get app info
+   * @return {String} appInfo
+   * @since 1.0.0
+   */
+  getAppInfo() {
```
Comment (popomore) [inline lib/loader/egg_loader.js]: antx 可以覆盖这个方法

**Comment by fengmk2** [comment]
+1

**Comment by popomore** [comment]
这个也算 minor 吧，增加了 root。

**Comment by popomore** [comment]
+ egg-core@1.1.0

**Pull request merged**